src/NzbDrone.Core/Download/ProcessDownloadDecisions.cs

```diff
@@ -74,6 +76,11 @@ public ProcessedDecisions ProcessDecisions(List<DownloadDecision> decisions)
                     _downloadService.DownloadReport(remoteEpisode);
                     grabbed.Add(report);
                 }
+                catch (ReleaseUnavailableException)
+                {
+                    _logger.Warn("Failed to download release from indexer, no longer available. " + remoteEpisode);
```

Comment:
Is there a reason not to use `_logger.Warn("Failed to download release from indexer, no longer available. {0}", remoteEpisode);`?